src/app/shell/admin-tools/data/admins/create-admin/create-admin.component.ts

```diff
@@ -40,14 +47,18 @@ const defaultValidators: ValidatorFn[] = [
 })
 export class CreateAdminComponent extends CreateFormComponent implements OnInit, OnDestroy {
   readonly validationConstants = ValidationConstants;
+  public readonly noSettlement = Constants.NO_SETTLEMENT;
+
   readonly phonePrefix = Constants.PHONE_PREFIX;
   readonly mailFormPlaceholder = Constants.MAIL_FORMAT_PLACEHOLDER;
   readonly adminsRole = AdminRoles;
 
   @Select(MetaDataState.institutions)
   institutions$: Observable<Institution[]>;
-  @Select(AdminState.selectedMinistryAdmin)
-  selectedMinistryAdmin$: Observable<MinistryAdmin>;
+  @Select(AdminState.selectedAdmin)
+  selectedAdmin$: Observable<BaseAdmin>;
```

Comment:
It would also be nice to add access modifiers to class fields

Reply:
added